docs/operator-manual/app-sync-using-impersonation.md

```diff
@@ -0,0 +1,114 @@
+# Application Sync using impersonation
+
+!!! warning "Alpha Feature"
+    This is an experimental, alpha-quality feature that allows you to control the service account used for the sync operation. The configured service account, could have lesser privileges required for creating resources compared to the highly privileged access required for the control plane operations.
+
+!!! warning
+    Please read this documentation carefully before you enable this feature. Misconfiguration could lead to potential security issues.
+
+## Introduction
+
+As of version 2.10, Argo CD supports syncing `Application` resources using the same service account used for its control plane operations. This feature enables users to decouple service account used for application sync from the service account used for control plane operations.
```

Comment:
Outdated version info. IMHO, we can remove it completely.

Reply:
Yes, will update the version.

Reply:
Updated the comments